content/docs/user-guide/gems/reference/aws/aws-gamelift/build-packaging-for-linux.md

```diff
@@ -58,40 +59,42 @@ You must create a separate installation folder to copy over the required assets,
 
     ```bash
     #!/bin/sh
-  
-    sudo cp -a ./lib64/* /lib64/.
-  
+
+    sudo cp -a ./lib64/libX* /lib64/.
+    sudo cp -a ./lib64/libbsd* /lib64/.
+
     if cat /etc/system-release | grep -qFe 'Amazon Linux release 2'
     then
-     sudo yum groupinstall 'Development Tools' -y
-     sudo yum install python3 -y
+    sudo yum update -y
```

Comment:
Did you intentionally remove the line indentation inside the `then` clause? Might want to put that back for readability. (And maybe use two spaces - I think it was only one space before.)

Reply:
Thanks for catching, I fixed the indentation!